src/styles/main.scss

```diff
@@ -0,0 +1,42 @@
+@import './utils/mixin';
+@import './utils/variables';
+
+.page {
+  justify-content: center;
+  align-items: center;
+  display: flex;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  width: ($cellSize * $weakSize) +6px;
+  padding-inline: $cellPading;
```

Comment:
The variable `$cellPading` appears to be misspelled. The correct spelling should be `$cellPadding` if it's meant to represent 'padding'.